DigiMeSDK/Core/Classes/Errors/NSError+Auth.m

```diff
@@ -55,7 +55,14 @@ + (NSError *)authError:(AuthError)authError reference:(nullable NSString *)error
     {
         NSMutableDictionary<NSErrorUserInfoKey, id> *userInfo = [NSMutableDictionary dictionary];
         NSString *errorDescription = [[self class] authDescription:authError];
-        userInfo[NSLocalizedDescriptionKey] = [NSString stringWithFormat:@"%@ %@ %@", errorDescription, NSLocalizedString(@"Reference:", nil), errorReference];
+        if (errorReference)
```

Comment:
The outer `if` statement is already checking `(errorReference != nil)`, so the inner check is redundant and the `else` will never be executed.